Makefile

```diff
@@ -16,3 +16,6 @@ benchmark:
 checks:
 	go vet ${PKGS}
 	staticcheck ${PKGS}
+
+lint:
+	golangci-lint run -c .golangci.yml
```

Comment:
BTW, a "foolproof" way to run this even if it's not installed in the user system (although could it be a bit slow) is to replace this with:
```suggestion
	go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run -c .golangci.yml
```

Reply:
I like it